core/server/services/slack.js

```diff
@@ -98,7 +98,7 @@ function listener(model, options) {
 
 function testPing() {
     ping({
-        message: 'Heya! This is a test notification from your Ghost blog :simple_smile:. Seems to work fine!'
+        message: 'Heya! This is a test notification from your Ghost blog :). Seems to work fine!'
```

Comment:
Tested with `:)` and this doesn't get rendered as emoji in Slack. I suggest, we change it to `:smile:` which - apparently - will work for both, Slack (I tested it and it works) and Discord.

Reply:
![image](https://user-images.githubusercontent.com/8037602/33816287-70bc3dcc-de6a-11e7-8887-37627ed13e42.png)

See here ^ first message with `:)` and second with `:smile:`.